src/igvfd/schemas/measurement_set.json

```diff
@@ -2,7 +2,7 @@
     "title": "Measurement Set",
     "$id": "/profiles/measurement_set.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "description": "Schema for a measurement set.",
+    "description": "A file set for a distinct measurement, such as the sequencing of a library. A measurement set contains the raw data gathered by the measurement.",
```

Comment:
"A file set for a distinct measurement, such as the sequencing of a genomic library generated by an experiment performed on a cell line. A measurement set contains the raw data files  generated by the measurement."